crates/util/python-utils/src/schnorr.rs

```diff
@@ -0,0 +1,62 @@
+use std::str::FromStr;
+
+use pyo3::{pyfunction, PyResult};
+use secp256k1::{schnorr::Signature, Keypair, Message, SecretKey, SECP256K1};
+use strata_crypto::{
+    sign_schnorr_sig as schnorr_sig_sign, verify_schnorr_sig as schnorr_sig_verify,
+};
+use strata_primitives::buf::{Buf32, Buf64};
+
+/// Signs a message using the Schnorr signature scheme.
+///
+/// This function generates a Schnorr signature for the given message using the provided secret key.
+/// The function returns the serialized signature and the corresponding public key.
+///
+/// # Arguments
+///
+/// * `message` - A string representing the message to sign, encoded in hexadecimal format.
+/// * `secret_key` - A string representing the secret key, encoded in hexadecimal format.
+///
+/// # Returns
+/// - The Schnorr signature, serialized and encoded in hexadecimal format.
+/// - The public key corresponding to the secret key, encoded in hexadecimal format.
```

Comment:
why using `-` here and `*` above?
```suggestion
/// * The Schnorr signature, serialized and encoded in hexadecimal format.
/// * The public key corresponding to the secret key, encoded in hexadecimal format.
```